case/src/main/kotlin/com/ritense/case/configuration/CaseAutoConfiguration.kt

```diff
@@ -83,4 +89,22 @@ class CaseAutoConfiguration {
     fun caseLiquibaseMasterChangeLogLocation(): LiquibaseMasterChangeLogLocation {
         return LiquibaseMasterChangeLogLocation("config/liquibase/case-master.xml")
     }
+
+    @Bean
+    fun enumDisplayTypeParameterType(): NamedType {
+        return NamedType(EnumDisplayTypeParameter::class.java, "enum")
+    }
+
+    @Bean
+    fun dateFormatDisplayTypeParameterType(): NamedType {
+        return NamedType(DateFormatDisplayTypeParameter::class.java, "dateFormat")
```

Comment:
shouldn't the string here just be `"date"` since it should match the types listed here: https://ritense.atlassian.net/wiki/spaces/RIT/pages/1101955073/Configurable+case+list+-+API+structure

Reply:
will resolve